data/card_catalogue/upgradeable-1.json

```diff
@@ -298,8 +382,9 @@
     "tier": "green",
     "details": {
       "rules": {
-        "passive": "Anything thrown on you is repelled with equal force",
-        "active": "Any damage dealt to you from the next attack targeting you is dealt both to you and your attacker",
+        "active": "The next amount of damage dealt to you is also dealt to your attacker.",
+        "passive": "Thrown objects bounce off of you."
```

Comment:
re-phrased this one, I originally had it 

> Objects collide with you elastically

but that might be a little too technical even if it's probably the most accurate and concise way to put it.

Reply:
resolved in ef33c754f7de80c1e2166505c2492983c098b1da